examples/src/main/scala/TracingExample.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import cats.effect.IO
+import cats.effect.IOApp
+import cats.effect.MonadCancelThrow
+import cats.effect.std.Console
+import cats.syntax.all._
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk
+import org.typelevel.otel4s.java.OtelJava
+import org.typelevel.otel4s.trace.Tracer
+
+trait Work[F[_]] {
+  def doWork: F[Unit]
+}
+
+object Work {
+  def apply[F[_]: MonadCancelThrow: Tracer: Console]: Work[F] =
+    new Work[F] {
+      def doWork: F[Unit] =
+        implicitly[Tracer[F]].span("Work.DoWork").use { span =>
+          span.addEvent("Starting the work.") *>
+            doWorkInternal *>
+            span.addEvent("Finished working.")
+        }
+
+      def doWorkInternal =
+        Console[F].println("Doin' work")
+    }
+}
+
+class TracingExample extends IOApp.Simple {
+  def getTracer: IO[Tracer[IO]] =
+    for {
+      otel4j <- IO(
+        AutoConfiguredOpenTelemetrySdk.builder.build().getOpenTelemetrySdk
```

Comment:
This can return a null that I swept under the rug...

Reply:
Now I can't find the reference to that null in the javadoc.  I thought it was when it was disabled...